src/lib/Libutil/misc_utils.c

```diff
@@ -1192,6 +1192,56 @@ break_comma_list(char *strlist)
 	return (break_delimited_str(strlist, ','));
 }
 
+/**
+ * @brief
+ *		find a string in a NULL terminated string array
+ *
+ * @param[in]	strarr	-	the string array to search
+ * @param[in]	str	-	the string to find
+ *
+ * @return	int
+ * @retval	1	: if the string is found
+ * @retval	0	: the string is not found or on error
+ *
+ */
+int
+find_string(char **strarr, char *str)
+{
+	int ind;
+
+	ind = find_string_ind(strarr, str);
+
+	if (ind >= 0)
+		return 1;
+
+	return 0;
+}
+
+/**
+ * @brief
+ * 		find index of str in strarr
+ *
+ * @param[in]	strarr	-	the string array to search
+ * @param[in]	str	-	the string to find
+ *
+ * @return	int
+ * @retval	index of string
+ * @retval	-1	: if not found
+ */
+int
+find_string_ind(char **strarr, char *str)
```

Comment:
Is this a static function to be used only by find_string()

Reply:
No, it is called directly as well by the scheduler